text/0757-default-modifier-manager.md

```diff
@@ -0,0 +1,276 @@
+ - The [implementation](gh-vm-1348) for [RFC #756](rfc-756)
+   (especially around where the default helper manager is chosen as the "fallback" when an existing manager is not found)
+ - The modifier-manager in [ember-could-get-used-to-this](ecgutt-modifier)
+   The main differences being that this RFC proposes
+   - no need to import anything or wrap in a function call
+   - instead of `(element, positionalArgs, namedArgs)` as the function signature, the signature
+     will be `(element, ...positionalArgs, namedArgs)`
+
+[gh-vm-1348]: https://github.com/glimmerjs/glimmer-vm/pull/1348
+[rfc-756]: https://github.com/emberjs/rfcs/pull/756
+[ecgutt-modifier]: https://github.com/tracked-tools/ember-could-get-used-to-this/blob/master/addon/-private/modifiers.js
+
+## How we teach this
+
+On the [Template Lifecycle, DOM, and Modifiers](tldm) page,
+we'll want to insert a section on custom modifiers towards the end, and maybe swap out the
+existing documentation referencing `ember-modifier` and/or `@ember/render-modifiers`.
+We'll also want examples of how to manage modifiers with tracked data, maybe when to use a
+local/private modifier, rather than something you'd put in `app/modifiers`.
+Maybe a whole page is added to the guides on modifiers, their philosophy, when to use them, etc.
+
+While the overall API for modifiers in general has been "not perfect", there has been a fair amount of
+experimentation with modifiers since the 3.11 - 3.16 (pre-octane) era. Function modifiers are
+significantly simpler than class-based modifiers, and the complexity of the specifics of class-
+based modifiers is where most of the hesitance around adding modifiers by default has been.
+
+Existing users of Ember should be made aware of these capabilities, once implemented, via
+the release blog post, with some examples -- but folks watching developments in the ecosystem
+will likely be aware of ember-could-get-used-to-this, which kind of implements
+some parts of this RFC, so the migration path for users of ember-could-get-used-to-this should
+be straight-forward.  That addon _may_ want to deprecate their similar functionality after
+the proposed behavior here lands -- though, this RFC suggests implementation such that developers
+may still use ember-could-get-used-to-this without disruption.
+
+There will also be a polyfill that implements a modifier-manager for `Function.prototype`, like
+the [helpers polyfill](helpers-polyfill) does for a helper-manager.
+
+[tldm]: https://guides.emberjs.com/release/components/template-lifecycle-dom-and-modifiers/
+[helpers-polyfill]: https://github.com/NullVoxPopuli/ember-functions-as-helper-polyfill
+
+## Drawbacks
+
+
+There could be some awkwardness around the last argument
+passed to the function, as the type signature may not match how the call-site expects
+the signature to be. Projects like [Glint](https://github.com/typed-ember/glint#readme)
+would be essential for helping with clarity here.
```

Comment:
I'm not clear what you mean by this! Can you explain a bit more?

Reply:
yea!